Source/Core/Core/HW/WiiSaveCrypted.cpp

```diff
@@ -189,7 +189,7 @@ void CWiiSaveCrypted::ReadHDR()
 	}
 	std::string banner_file_path = m_wii_title_path + "banner.bin";
 	if (!File::Exists(banner_file_path) ||
-		AskYesNoT("%s already exists, overwrite?", banner_file_path.c_str()))
+		AskYesNoT("%s already exists. You can still backup the current save files before overwriting.\nOverwrite now?", banner_file_path.c_str()))
```

Comment:
You're right. I felt that users working with save files knew whether they wanted to backup the files or not, and that the previous message either served as a reminder or a nag. I'm happy to make the message more encouraging. Something like "Consider making a backup of...".